### PR TITLE
adding gear ratios to make accurate rpm

### PR DESCRIPTION
**Pull request by LandonBayer**

### CODE REVIEW

File: constants.py [hunk 1]
```diff
@@ -666,6 +666,9 @@
 kIntakeCameraHeightInMeters = 0.5
 kIntakeCameraCenterOffsetInMeters = 0.2
 kIsIntakeCameraCentered = False
+kIntakeGearRatio = 36 / 24
+kIndexerGearRatio = 5 / 1
+kStagingGearRatio = 5 / 1
 
 kIntakeSpeed = 500
```
Comment (lmaxwell24) [inline constants.py:673]: Speeds should be changed to reflect new changes

**Review by lmaxwell24 — CHANGES_REQUESTED** [review]

File: constants.py (hunk 1 above)
Comment (LandonBayer) [reply constants.py:673]: They're changed in the set

Comment (lmaxwell24) [reply constants.py:673]: right, but the speeds on the intake should be multiplied by like 6 and indexer/intake by 1.25 or so to account for all the speed changes to remain constant speeds from what we set

Comment (LandonBayer) [reply constants.py:673]: What do you mean by "account for all the speed changes to remain constant speeds"? These make our constants, which are measured in rpm, to be the output rpm. That's what we wanted originally, no?

**Review by caloisio — APPROVED** [review]

**Review by caloisio — APPROVED** [review]

**Review by ericyamaguchi — APPROVED** [review]

**Pull request merged**